src/index.html

```diff
@@ -10,7 +10,33 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="big-form"
+    >
+      <input
+        type="search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-field search-field--big icon icon--big"
```

Comment:
Why do you need such a lot of classes here? It's not an icon, as I see
```suggestion
        class="search-field search-field--big"
```